gradle/wrapper/gradle-wrapper.properties

```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://nexus.arin.net/repository/gradle-distributions/gradle-6.5-all.zip
```

Comment:
Good catch.  Surprised we never had someone comment on this before.

Reply:
Actually I inadvertently ended up with a copy-n-paste mistake while updating Gradle version. :) It has since been corrected to a non-nexus, public URL.